.github/workflows/ci.yml

```diff
@@ -496,6 +496,7 @@ jobs:
     with:
       project-name: Component[OpenTelemetry.Resources.OperatingSystem]
       code-cov-name: Resources.OperatingSystem
+      os-list: '[ "windows-latest", "ubuntu-latest", "macos-latest" ]'
```

Comment:
You can try this https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1964#issuecomment-2234020323